pkg/hostagent/agent.go

```diff
@@ -138,6 +140,7 @@ func NewHostAgent(config *HostAgentConfig, env Environment, log *logrus.Logger)
 				DefaultNs,
 			},
 		},
+		endPointSliceEnabled: false,
```

Comment:
This should come from config, right? You can default the config to false.

Reply:
Yes Joji, i will change this code reading it from config